modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/visualization_modes/list/List.tsx

```diff
@@ -3,12 +3,140 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
+import ClayAlert from '@clayui/alert';
+import {ClayButtonWithIcon} from '@clayui/button';
+import {ClayInput} from '@clayui/form';
+import ClayLayout from '@clayui/layout';
+import ClayTable from '@clayui/table';
+import classNames from 'classnames';
+import {openModal} from 'frontend-js-web';
 import React from 'react';
 
+import '../../../../css/ListVisualizationMode.scss';
+import {IFDSViewSectionProps} from '../../../FDSView';
 import {IBaseVisualizationMode} from '../VisualizationModes';
+import {IFDSField} from '../table/Table';
+import AddFieldsModalContent from '../table/modal_content/AddFieldsModalContent';
 
 export interface IList extends IBaseVisualizationMode<'list'> {}
 
-export default function List() {
-	return <>List</>;
+interface IListField {
+	fieldId: string;
+	label: string;
+}
+
+const LIST_VISUALIZATION_MODE_FIELDS: IListField[] = [
+	{fieldId: 'title', label: Liferay.Language.get('title')},
+	{fieldId: 'description', label: Liferay.Language.get('description')},
+	{fieldId: 'symbol', label: Liferay.Language.get('symbol')},
+	{fieldId: 'link', label: Liferay.Language.get('link')},
+	{fieldId: 'label', label: Liferay.Language.get('label')},
+];
+
+export default function List(props: IFDSViewSectionProps) {
+	const [fieldValues, setFieldValues] = React.useState<
+		Record<IListField['fieldId'], IFDSField>
+	>({});
+
+	return (
+		<ClayLayout.ContentCol className="c-gap-4">
+			<ClayAlert
+				displayType="info"
+				title={`${Liferay.Language.get('info')}:`}
+				variant="stripe"
+			>
+				{Liferay.Language.get(
+					'list-visualization-mode-wont-be-shown-until-you-assign-at-least-one-field-to-a-list-element'
+				)}
+			</ClayAlert>
+
+			<ClayTable className="mb-0">
+				<ClayTable.Head>
+					<ClayTable.Cell className="list-visualization-mode-label-cell">
```

Comment:
This bit seems to be causing a warning in console log:
![image](https://github.com/liferay-frontend/liferay-portal/assets/5435511/5e50fb94-ebb7-4e05-b422-4c694cf51f6e)

Reply:
ups, right, that should be a th...